### DISCUSSION TITLE
area bug, problem

### DISCUSSION THREAD

**Discussion by Tenson443** [ppy/osu#29539] [Q&A]
when i try to use my own area(open tablet driver) my mouse just doesnt move to left side
![image](https://github.com/user-attachments/assets/2ba869c8-e6e4-4563-9fd7-ca2d07f62265)
i dont want to use osu area because u dont have any plugins here :(

**Comment by Joehuu** [comment]
Does disabling high precision mouse fix it? You would need to edit your area (in OTD) though because you changed sensitivity in-game (disabling high precision mouse disables sensitivity).

Also are you using multiple monitors? Testing briefly, it seems to break the area unless I change the display area (in OTD) to my non-primary monitor, which is weird.
